TaskWithClass/MyString.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <string>
+#include <iostream>
+
+class String {
```

Comment:
Either name your class MyString or rename the file to String.h. I'm rooting for the first.